ui-ngx/src/app/modules/home/components/device/device-credentials.module.ts

```diff
@@ -22,14 +22,16 @@ import { DeviceCredentialsComponent } from '@home/components/device/device-crede
 import { DeviceCredentialsLwm2mComponent } from '@home/components/device/device-credentials-lwm2m.component';
 import { DeviceCredentialsLwm2mServerComponent } from '@home/components/device/device-credentials-lwm2m-server.component';
 import { DeviceCredentialsMqttBasicComponent } from '@home/components/device/device-credentials-mqtt-basic.component';
+import { DeviceExampleCommandComponent } from '@home/components/device/device-example-command.component';
 
 @NgModule({
   declarations: [
     CopyDeviceCredentialsComponent,
     DeviceCredentialsComponent,
     DeviceCredentialsLwm2mComponent,
     DeviceCredentialsLwm2mServerComponent,
-    DeviceCredentialsMqttBasicComponent
+    DeviceCredentialsMqttBasicComponent,
+    DeviceExampleCommandComponent
```

Comment:
This improvement work in other PR: https://github.com/thingsboard/thingsboard/pull/8803

Reply:
Will use device api after it will be implemented